### ISSUE TITLE
Improve function docstring for Sphinx 

### ISSUE THREAD

**Issue by paugier** [serge-sans-paille/pythran#794]
Building documentation with Sphinx of a project containing Pythran functions leads to rst warnings ("WARNING: Definition list ends without a blank line; unexpected unindent.") because there is no line between the supported protocols and the function docstring. For example:

```
Supported prototypes:
    - compute_increments_dim1(float64[][], int)
    - compute_increments_dim1(float64[][].T, int)
Compute the increments of var over the dim 1.

Blablabla...
```

One more '\n' would be useful:

```
Supported prototypes:
    - compute_increments_dim1(float64[][], int)
    - compute_increments_dim1(float64[][].T, int)

Compute the increments of var over the dim 1.

Blablabla...
```

It would be even better to put the first line of the docstring (which has a special meaning for Sphinx) before the supported prototypes...

```
Compute the increments of var over the dim 1.

Supported prototypes:
    - compute_increments_dim1(float64[][], int)
    - compute_increments_dim1(float64[][].T, int)

Blablabla...
```

**Comment by serge-sans-paille** [comment]
PR are welcome, see https://github.com/serge-sans-paille/pythran/blob/master/pythran/toolchain.py#L132 and https://github.com/serge-sans-paille/pythran/blob/master/pythran/toolchain.py#L170 :-)

**Comment by serge-sans-paille** [comment]
Fixed by @paugier in 1c64ee3c57f64b23611fe01d14db60ee45f2cf04
